ruby_review_week_four.rb

```diff
@@ -2,36 +2,170 @@
 
 # 1. What is a variable?
 
+# it is a named location in memory to store data via assignment (variable = some_data) that can be later called upon with that name to return what is stored there.
+
 # 2. What Ruby method can you use to make a loop from an array?
 
+# Array.each
+
 # 3. In Ruby, what are booleans?
 
+# a data type that only has two possible values, true and false
+# the result of some condition can also return true or false e.g. "if (10 > 9)"" is the same as writing "if true"
+
 # 4. How do classes relate to objects?
 
+# every variable is an object of a certain class type. in a variable we store an instance of an object. the object is created calling the constructor of a class in some way.
```

Comment:
this is correct, i also like the answer of classes are `blueprints` or `templates` for objects